test/powershell/Modules/Microsoft.PowerShell.Utility/Out-File.Tests.ps1

```diff
@@ -127,4 +127,10 @@ Describe "Out-File" -Tags "CI" {
 	# reset to not read only so it can be deleted
 	Set-ItemProperty -Path $testfile -Name IsReadOnly -Value $false
     }
+    It "Should be able to use the 'Path' alias for the 'FilePath' parameter" {
+    {Out-File -Path $testfile -InputObject $expectedContent | Should Not Throw}
```

Comment:
Right pattern here is `{ #your code } | Should Not Throw`.
You can find multiple samples in the same Out-File.Tests.ps1 file.
(`Describe`, `Context`, `It` and `Should` is Pester's functions. So you send your test code to a Pester function as script block parameter, Pester execute it and piping _result of your script block_ to `Should` to check result - ok or fail.)